apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/VedtakBehandlingService.kt

```diff
@@ -647,6 +651,7 @@ class VedtakBehandlingService(
                         periode = Periode(opphoerFraOgMed, null),
                         beloep = null,
                         type = UtbetalingsperiodeType.OPPHOER,
+                        regelverk = Regelverk.fraDato(opphoerFraOgMed.atDay(1)),
```

Comment:
Får ingen perioder for dette fra beregning, så må legge til regelverk her

Reply:
Tenker kommentaren din på github her hadde vært nice å hatt i koden også :p